### ISSUE TITLE
Ops for CalibratedSpace

### ISSUE THREAD

**Issue by stelfrich** [imagej/imagej-ops#381]
There is no convenient way to compute statistics using e.g. `Ops.Geometric.Size` for `CalibratedSpace`.

Should Ops that compute stats for calibrated data be added on a per case basis for now? Does it even make sense to approach this issue before the data model has undergone the announced re-structuring?

**Comment by ctrueden** [comment]
Yeah, @rimadoma ran into this same issue when trying to port some BoneJ1 functionality to Ops. But I think we need to wait for the redesign of `ImgPlus` so that we don't waste a lot of effort. Hopefully this fall we can push on it—at minimum, we can make a plan during the summer Konstanz hackathon.

**Comment by rimadoma** [comment]
Well I've started writing some small [utility functions](https://github.com/rimadoma/MadisonHacks/blob/master/utilities/src/main/java/org/bonej/utilities/CalibratedAxisUtil.java) for AnnotatedSpace / CalibratedAxis, but that's about it
